Makefile

```diff
@@ -199,7 +199,7 @@ e2e-cross: e2e-linux-amd64 e2e-darwin-amd64 e2e-windows-amd64.exe
 
 .PHONY: checksum
 checksum:
-	for f in out/localkube out/minikube-linux-amd64 out/minikube-darwin-amd64 out/minikube-windows-amd64.exe out/minikube.iso; do \
+	for f in out/localkube out/minikube-$(GOOS)-$(GOARCH)$(IS_EXE) out/minikube.iso; do \
```

Comment:
This line is part of the release build, it tries to checksum all known binaries (they have been cross-compiled earlier).

Thus it doesn't make sense to checksum only one of them. Should probably make the list into a make variable though.